src/components/SchemaViewer.svelte

```diff
@@ -11,24 +11,38 @@
       .split(" ")
       .filter((t) => t.length > 0);
 
-    const addVisibility = (node) => {
+    const addVisibility = (node, parentNodeNames = [`${node.name}`]) => {
```

Comment:
```suggestion
    const addVisibility = (node, parentNodeNames = ["__root__"]) => {
```

Starting with the name of the current node means that there are a lot of changes from the last commit. My last suggestion was to start with a literal root value (e.g. `__root__` to make it obvious that it's not an actual column name). This way, there wouldn't be any reason to check for whether `parentNodeNames` was empty, like in this block:

https://github.com/mozilla/glean-dictionary/pull/84/commits/22c7a2659d4ffe75c56eb74b222af4d244d20bb1#diff-813bcbaca6a1e7fb8e6fc6ab82618d42ae769b8beb12d76f4582c08ecfe8f1d1R22-R26